includes/actions/remove-shipping.php

```diff
@@ -62,7 +62,9 @@ protected function edit_subscription( $shipping_data, $subscription ) {
 
 		foreach ( $subscription->get_shipping_methods() as $line_item ) {
 			// Same approach used in Abstract_WC_Order::has_shipping_method() to check for method
-			if ( 0 === strpos( $line_item->get_method_id(), $shipping_data['shipping_method_id'] ) ) {
+			if ( 0 === strpos( $line_item->get_method_id(), $shipping_data['shipping_method_id'] )
```

Comment:
Minor: If we're going to split the `if` statement across multiple lines, then the first condition should be on its own line.